src/cmake/llvm_macros.cmake

```diff
@@ -0,0 +1,128 @@
+# Copyright Contributors to the Open Shading Language project.
+# SPDX-License-Identifier: BSD-3-Clause
+# https://github.com/AcademySoftwareFoundation/OpenShadingLanguage
+
+
+function ( EMBED_LLVM_BITCODE_IN_CPP src_list suffix output_name list_to_append_cpp extra_clang_args)
```

Comment:
It would be good to unify this with `MAKE_CUDA_BITCODE` at some point, since they're largely identical. And to generalize to other targets like SPIR-V. But that can wait until we're compiling for other non-host targets.

Reply:
Agreed, they are very similar, but our ability to test the CUDA path is limited and we don't want to break it by accident, or create merge issues for any changes in the MAKE_CUDA_BITCODE that might happen.  Maybe an interim solution would be to move both macros to a common file, so at least there right next to each other for reference.

Reply:
I think this is just fine for now. It'll be something to keep in mind when the time comes to add `MAKE_SPIRV_BITCODE`, or whatever the next target might be.